packages/cli/source/project/index.ts

```diff
@@ -3,10 +3,12 @@ import * as path from "path";
 import * as fsExtra from "fs-extra";
 import TaskList from "listr";
 import chalk from "chalk";
+import { chdir } from "process";
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
```

Comment:
You need to use import statements. I've already told you this.

Reply:
You can break down `const exec = util.promisify(require("child_process").exec);` into multiple statements, that's alright.